src/AppRoutes.tsx

```diff
@@ -46,6 +47,7 @@ export const AppRoutes = () => (
     {/* entity pages */}
     <Route path="products" element={<ProductsPage />} />
     <Route path="projects" element={<ProjectsPage />} />
+    <Route path="projects/:id" element={<ProjectDetailPage />} />
```

Comment:
@jakubvanko @jomrazek WDYT about using nested routes in cases like this? https://reactrouter.com/docs/en/v6/getting-started/concepts#nested-routes

Reply:
I believe this is exact place to use them :+1:

Reply:
Personally, I think that we should update all of the paths at once since there are other candidates for such a change (we already have admin pages admin/demo, admin/variables, admin/administration) and I believe that mixing two approaches could be confusing.

Reply:
@jakubvanko yes, all paths will be already udpated (it's part of the PR 62), the best will be wait once PR 62 is merged and rebase this one on top of it

Reply:
Okay, will change it once the PR62 is merged